backbone/src/main/res/layout/rsb_step_layout_active_step.xml

```diff
@@ -25,4 +25,22 @@
         android:textSize="@dimen/rsb_step_layout_countdown_text_size"
         android:textColor="@color/rsb_countdown_step_layout_text_color"/>
 
+    <ProgressBar
+        style="@style/Base.Widget.AppCompat.ProgressBar"
+        android:id="@+id/rsb_active_step_layout_progress"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:padding="@dimen/rsb_padding_large"
+        android:indeterminate="true"
+        android:visibility="gone"/>
+
+    <android.support.v7.widget.AppCompatImageView
```

Comment:
No, I don't think you do.  They wrote all the compat view to magically inject themselves at runtime in place of their legacy counterparts.  However, I've read a few posts that on certain OS' or maybe it was built tools(?) that it didn't work correctly.  Either way, you are right, in xml I don't have to specify this, so I am going to switch it back to ImageView